.rubocop.yml

```diff
@@ -84,4 +92,7 @@ Metrics/ModuleLength:
 Metrics/ClassLength:
   Enabled: false
 
+Style/CombinableLoops:
+  Enabled: false
```

Comment:
This thought a couple loops could be combined, but it would've taken a bigger refactor, not worth it.